ray-on-gke/user/kuberay-tpu-webhook/deployments/webhook-cfg.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: admissionregistration.k8s.io/v1
+kind: MutatingWebhookConfiguration
+metadata:
+  name: kuberay-tpu-webhook-cfg
+  annotations:
+    cert-manager.io/inject-ca-from: default/kuberay-tpu-webhook-certs
+webhooks:
+  - name: kuberay-tpu-webhook.default.svc
+    admissionReviewVersions: [v1]
+    sideEffects: NoneOnDryRun
+    clientConfig:
+      service:
+        name: kuberay-tpu-webhook
+        namespace: default
+        path: /inject
+    rules:
+      - operations: ["CREATE"]
+        apiGroups: ["*"]
+        apiVersions: ["*"]
```

Comment:
This seems fine if the webhook supports all versions, does it work for both v1alpha1 and v1?

Reply:
Yes, the webhook supports both v1alpha1 and v1.